services/metadactyl-clj/src/metadactyl/routes/analyses.clj

```diff
@@ -45,35 +45,35 @@
           :return      AnalysisUpdateResponse
           :summary     "Update an Analysis"
           :description       "This service allows an analysis name or description to be updated."
-          (service/coerced-trap uri AnalysisUpdateResponse
+          (service/coerced-trap nil AnalysisUpdateResponse
                                 apps/update-job current-user analysis-id body))
 
-  (DELETE* "/:analysis-id" [:as {:keys [uri]}]
+  (DELETE* "/:analysis-id" []
            :path-params [analysis-id :- AnalysisIdPathParam]
            :query       [params SecuredQueryParams]
            :summary     "Delete an Analysis"
            :description       "This service marks an analysis as deleted in the DE database."
-           (service/trap uri apps/delete-job current-user analysis-id))
+           (ok (apps/delete-job current-user analysis-id)))
 
-  (POST* "/shredder" [:as {:keys [uri]}]
+  (POST* "/shredder" []
          :query   [params SecuredQueryParams]
          :body    [body AnalysisShredderRequest]
          :summary "Delete Multiple Analyses"
          :description   "This service allows the caller to mark one or more analyses as deleted
          in the apps database."
-         (service/trap uri apps/delete-jobs current-user body))
+         (ok (apps/delete-jobs current-user body)))
 
-  (GET* "/:analysis-id/parameters" [:as {:keys [uri]}]
+  (GET* "/:analysis-id/parameters" []
         :path-params [analysis-id :- AnalysisIdPathParam]
         :query       [params SecuredQueryParams]
         :return      AnalysisParameters
         :summary     "Display the parameters used in an analysis."
         :description       "This service returns a list of parameter values used in a previously
         executed analysis."
-        (service/coerced-trap uri AnalysisParameters
+        (service/coerced-trap nil AnalysisParameters
```

Comment:
The name, `coerced-trap` probably doesn't fit anymore. We could probably change it to `coerce-response` and remove the error handling.